src/nrniv/nrnclass.h

```diff
@@ -3,9 +3,9 @@
 , Shape_reg(), PlotShape_reg(), PPShape_reg(), RangeVarPlot_reg(), SectionBrowser_reg(),
     MechanismStandard_reg(), MechanismType_reg(), NetCon_reg(), LinearMechanism_reg(), KSChan_reg(),
     Impedance_reg(), SaveState_reg(), BBSaveState_reg(), FInitializeHandler_reg(),
-    StateTransitionEvent_reg(), nrnpython_reg()
+    StateTransitionEvent_reg(), nrnpython_reg(), NMODLRandom_reg()
 #if USEDASPK
-                                    ,
+                                                     ,
```

Comment:
why extra line

Reply:
leave as is. just a reformatting of the where the comma is.( because of the increased length of line 6?